modules/vba_documents/app/swagger/vba_documents/document_upload/status_code_description.md

```diff
@@ -7,5 +7,6 @@ Unambiguous status code. Only present if status = "error"
 * `DOC105` - Invalid or unknown id
 * `DOC106` - File size limit exceeded. Each document may be a maximum of 100MB.
 * `DOC107` - Empty payload.
+* `DOC108` - Maxium dimensions exceeded. Height and width must be less than 21 in x 21 in.
```

Comment:
Was this ever decided on as a breaking change or not? Before we merge we need to confirm with Andrew on whether or not VetPro and Vetrespec are okay with this.

Reply:
Yes in the ticket they mentioned the communication

Reply:
it's not a breaking change